### PR TITLE
`Int::times`: clean nitunit and expose negative behavior

### PR DESCRIPTION
**Pull request by Morriar**
I tried to use `Int::times` but the nitunit wasn't clear enough on the expected behavior.

Thanks at @BlackMinou for is moral support on this.

### CODE REVIEW

File: lib/standard/collection/range.nit [hunk 1]
```diff
@@ -240,12 +240,8 @@ end
...
 	# Returns the range from 0 to `self-1`, is used to do:
 	#
-	#     var s = new Array[String]
-	#     for i in 3.times do s.add "cool"
-	#     assert s.join(" ") == "cool cool cool"
-	#
-	#     s.clear
-	#     for i in 10.times do s.add(i.to_s)
-	#     assert s.to_s == "0123456789"
+	#     assert 3.times == [0..3[
```
Comment (privat) [inline lib/standard/collection/range.nit]: It is used to do asserts?

I suggest that you also update the synopsis.

Comment (Morriar) [inline lib/standard/collection/range.nit]: Damn, the synopsis was crap too? I will change that.

**Comment by Morriar** [comment]
Reroll I changed the synopsis and kept the previous nitunit as a loop example.

(also kept the reference to Abed)

**Comment by jpages** [comment]
+1

**Comment by privat** [comment]
+1

**Pull request merged**